src/main/java/io/cdap/plugin/google/sheets/sink/GoogleSheetsSinkConfig.java

```diff
@@ -167,7 +167,7 @@ public class GoogleSheetsSinkConfig extends GoogleInputSchemaFieldsUsageConfig {
    * @param schema the schema to check compatibility
    */
   public void validate(FailureCollector collector, Schema schema) {
-    super.validate(collector);
+    super.getValidationResult(collector);
```

Comment:
should we do `collector.getOrThrowException` after this call everywhere?

Reply:
This will allow us to fail early, if the initial validation fails, there is no point of going forward